### ISSUE TITLE
Error in GUI!

### ISSUE THREAD

**Issue by neverh1t** [ioBroker/ioBroker.admin#1614]
**Describe the bug** 

Error in GUI!
Please open the browser console (F12), copy error text from there and create the issue on [github](https://github.com/ioBroker/ioBroker.admin/issues)
Without this information it is not possible to analyse the error.
It should looks like
```
instrument.ts:124 TypeError: Cannot read properties of undefined (reading 'value')
    at vn.renderItem (ConfigText.jsx:36:34)
    at vn.render (ConfigGeneric.jsx:514:39)
    at Wi (react-dom.production.min.js:204:189)
    at Rs (react-dom.production.min.js:203:149)
    at oa (react-dom.production.min.js:292:172)
    at ra (react-dom.production.min.js:280:389)
    at Jc (react-dom.production.min.js:280:320)
    at ml (react-dom.production.min.js:280:180)
    at iu (react-dom.production.min.js:271:88)
    at bs (react-dom.production.min.js:268:429)
    at Ln (scheduler.production.min.js:13:203)
    at MessagePort.Gn (scheduler.production.min.js:14:128)
```

**Comment by arteck** [comment]
@neverh1t really ??? there is a very accurate description


version 6.2.6

https://user-images.githubusercontent.com/6681528/181691045-2874382a-a0d9-4069-bb25-2f59f3d70637.mp4

![grafik](https://user-images.githubusercontent.com/6681528/181691070-048966ba-52bc-4044-8ddd-7eb8bb7cf116.png)

**Comment by JohGrei** [comment]
Habe den selben Fehler. Beim Öffnen der "custom settings" eines Attributes schmeissts die Seite weg. Der Stacktrace gibt für mich aber nicht viel her. Habe auch den Influx Adapter installiert.

Admin Version 6.2.6

`TypeError: Cannot read properties of undefined (reading 'value')
    at vn.renderItem (ConfigText.jsx:36:34)
    at vn.render (ConfigGeneric.jsx:514:39)
    at Wi (react-dom.production.min.js:204:189)
    at Rs (react-dom.production.min.js:203:149)
    at oa (react-dom.production.min.js:292:172)
    at ra (react-dom.production.min.js:280:389)
    at Jc (react-dom.production.min.js:280:320)
    at ml (react-dom.production.min.js:280:180)
    at iu (react-dom.production.min.js:271:88)
    at bs (react-dom.production.min.js:268:429)
    at Ln (scheduler.production.min.js:13:203)
    at MessagePort.Gn (scheduler.production.min.js:14:128)`

**Comment by Feuer-sturm** [comment]
Is fixed with Admin 6.2.7
